src/main/kotlin/it/unibo/alchemist/config/EngineMode.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2010-2023, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.config
+
+/**
+ * Engine modes.
+ * @property code mode code
+ */
+enum class EngineMode(val code: String) {
```

Comment:
Removed, see general changes to Loader and Launcher creation.

Engine config will be implemented with a similar mechanism in the PR for batch engine as currently it would do nothing.